Dockerfile

```
@@ -1,5 +1,31 @@
-FROM python:3.6-stretch
-MAINTAINER Shane Frasier <jeremy.frasier@trio.dhs.gov>
+
+FROM python:3.6.14-stretch
```

Forgot to take this into account in my previous suggestion.

```suggestion
FROM python:3.6.14-stretch
```

---

I'm lost.  What is the difference you are asking for?

---

Ah, now I see you're removing the blank line at the top.  See commit e465fdd.

---

Just a blank line at the top of the file that I forgot to include in https://github.com/cisagov/trustymail_reporter/pull/44#discussion_r671038022.